base/missing.jl

```diff
@@ -86,6 +86,8 @@ isequal(::Any, ::Missing) = false
 isless(::Missing, ::Missing) = false
 isless(::Missing, ::Any) = false
 isless(::Any, ::Missing) = true
+isless(::Missing, ::Nothing) = true
```

Comment:
Shouldn't this method be covered by `isless(::Any, ::Nothing)`?

Reply:
It's here to resolve a method ambiguity between `isless(::Missing, ::Any)` and `isless(::Any, ::Nothing)`.